src/components/Modals/DailyLimitModal/DailyLimitModal.scss

```diff
@@ -1,33 +1,3 @@
 .daily-limit-modal {
-    position: absolute;
-    top: 50%;
-    left: 50%;
-    right: auto;
-    bottom: auto;
-    margin-right: -50%;
-    transform: translate(-50%, -50%);
-    width: 44rem;
-    padding: 2.4rem;
-    border-radius: 8px;
-    background: #fff;
-    box-shadow: 0px 32px 64px 0px rgba(14, 14, 14, 0.14);
-
-    @include mobile {
-        padding: 1.6rem;
-        width: 32.8rem;
-    }
-
-    &__text {
-        margin: 2.4rem 0;
-
-        @include mobile {
-            margin: 1.6rem 0;
-        }
-    }
-
-    &__footer {
-        display: flex;
-        justify-content: flex-end;
-        gap: 1rem;
-    }
+    @include default-modal;
```

Comment:
It is recommended not to use @include in this case. But, Using @include will duplicate the CSS rules every time it is called, which can lead to larger CSS files and potential performance issues. Instead, consider sharing class names or using @extend. @extend will merge the styles without duplicating them, making the CSS more efficient and easier to maintain.


<img width="1050" alt="Screenshot 2024-07-04 at 11 10 18 AM" src="https://github.com/deriv-com/p2p/assets/126637868/a0b9cbd6-45d3-4c66-9fdf-9f842acd62d1">

Reply:
@extend:

<img width="1042" alt="Screenshot 2024-07-04 at 11 15 45 AM" src="https://github.com/deriv-com/p2p/assets/126637868/23e76319-ece8-45a0-b86f-3e2fff8c00bf">

Reply:
this change would require the mixin to be changed to placeholder and would affect all the other modals. can take it up separately. 👍